docs/specs/component.md

```diff
@@ -205,53 +199,25 @@ sending it, like the `prometheus_exporter` sink, SHOULD NOT publish this metric.
     defined properties as metric tags.
 * Logs
   * MUST log a `Bytes sent.` message at the `trace` level with the
-    defined properties as key-value pairs. It MUST NOT be rate limited.
+    defined properties as key-value pairs.
+  * MUST NOT be rate limited.
+
+#### ComponentError
 
-#### Error
+**Extends the [Error event].**
 
-*All components* MUST emit error events when an error occurs, and errors MUST be
-named with an `Error` suffix. For example, the `socket` source emits a
-`SocketReceiveError` representing any error that occurs while receiving data off
-of the socket.
+*All components* MUST emit error events in accordance with the [Error event]
+requirements.
```

Comment:
Shifted the `*Error` event requirements to the instrumentation spec since this is shared across Vector.